web/templates/input_pill.hbs

```diff
@@ -16,7 +19,6 @@
             {{ display_value }}
         </span>
         {{~#if should_add_guest_user_indicator}}&nbsp;<i>({{t 'guest'}})</i>{{~/if~}}
-        {{~#if deactivated}}&nbsp;({{t 'deactivated'}}){{~/if~}}
```

Comment:
Are users the only input pills that got a `deactivated` variable? How do you check?

Reply:
Seems OK:
```
    return render_input_pill({                                                                          
        has_stream: true,                                                                               
        stream,                                                                                         
        display_value: get_display_value_from_item(item),                                               
        stream_id: item.stream_id,                                                                      
    });                                                                                                 
```